src/components/tab-nav/tab-nav.tsx

```diff
@@ -105,6 +106,11 @@ export class TabNav {
 
   connectedCallback(): void {
     this.parentTabsEl = this.el.closest("calcite-tabs");
+    this.resizeObserver?.observe(this.el);
+  }
+
+  disconnectedCallback(): void {
+    this.resizeObserver?.unobserve(this.el);
```

Comment:
I wonder if we should just call `this.resizeObserver?.disconnect()` since we're only observing 1 element anyway. I guess it doesn't matter either way.

Reply:
Good point. I'll update it for consistency.